### PR TITLE
Explain purpose

### PR DESCRIPTION
**Pull request by warpech**
Implements https://github.com/StarcounterAppsFreezer/UsageTracker/issues/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Hosts usage trackers like Google Analytics. It is like [Tag Manager](https://www.google.com/analytics/tag-manager/) for Starcounter apps.
 
+## Purpose of this app
+
+This micro app solves a problem of dynamically adding a structured snippet of code (such as Google Analytics tracking script) to a system of apps, without having to alter the code of any of the existing apps.
```
Comment (Mackiovello) [inline README.md]: solves a problem -> solved the problem

adding a structure snippet -> adding structured snippets

Comment (warpech) [inline README.md]: Why past tense in "solved the problem"? Can I rephrase that to "This micro app allows to dynamically add ..."

Comment (Mackiovello) [inline README.md]: That's my bad, it should be "solves the problem"

Comment (Mackiovello) [inline README.md]: Your suggestion works too

**Comment by warpech** [comment]
Fixed, thanks to the review!

**Pull request merged**